test/verify/test_isnan.cpp

```diff
@@ -0,0 +1,21 @@
+#include <limits>
+#include "verify_program.hpp"
+#include <migraphx/program.hpp>
+#include <migraphx/generate.hpp>
+#include <migraphx/make_op.hpp>
+
+struct test_abs : verify_program<test_abs>
```

Comment:
You can also add a test for one of the nonstandard shapes e.g. broadcast, slice, or transpose.